.rspec

```diff
@@ -1 +1,2 @@
---color
\ No newline at end of file
+--color
+--require spec_helper
```

Comment:
hell no... I don't like all that messy text in my console output. Stack trace is more that enough